src/pages/EventAdministration/components/EventEditor.tsx

```diff
@@ -467,6 +467,7 @@ const EventEditor = ({ eventId, goToEvent }: EventEditorProps) => {
           </Row>
           <Bool
             control={control}
+            disabled={!(data?.list_count && data?.list_count > 0)}
```

Comment:
Jeg tror dette fungerer altså, men kanskje det er noe typescript wodo som foregår
```suggestion
            disabled={data?.list_count && data.list_count > 0}
```

Reply:
Ja funker nok, men TS klager på det hvis jeg ikke tar ! foran

Reply:
Det å ta utropstegn forran gjør akkuratt motsatt av det vi ønsker. Nå vil den være disabled når vi ikke har deltagere, mens enabled når vi har deltagere

Reply:
Jupp har fiksa nå